src/main/resources/config/application.yml

```diff
@@ -15,6 +15,24 @@ spring:
   resources:
     static-locations: file:web
 
+# metrics
+management:
+  metrics:
+    export.prometheus.enabled: true
+    distribution:
+      percentiles:
+        http.server.requests: 0.5, 0.75, 0.95, 0.99
+  endpoint:
+    metrics.enabled: true
+    prometheus.enabled: true
+    heapdump.enabled: false
+    health:
+      show-details: always
+    env:
+      keys-to-sanitize: .*password.*
```

Comment:
Why do we need to customize this setting?

Reply:
Yanagishima config has `hive.jdbc.password.{datasource}` property, but default `keys-to-sanitize` expects "ending" with `password`. We can see raw password in `/actuator/env` endpoint without this customization. 
https://docs.spring.io/spring-boot/docs/2.0.x/reference/html/howto-actuator.html#howto-sanitize-sensible-values